Assets/MixedRealityToolkit/StandardAssets/Shaders/MixedRealityStandard.shader

```diff
@@ -319,8 +344,11 @@ Shader "Mixed Reality Toolkit/Standard"
 #if defined(_VERTEX_COLORS)
                 fixed4 color : COLOR0;
 #endif
+#if defined(_SPHERICAL_HARMONICS)
+                fixed3 ambient : COLOR1;
+#endif
 #if defined(_IRIDESCENCE)
-                fixed3 iridescentColor : COLOR1;
+                fixed3 iridescentColor : COLOR2;
```

Comment:
Does moving this from COLOR1 to COLOR2 have any impact on pre-existing materials?

Reply:
It does not since this is just changing the interpolator semantic which doesn't have a connection to serialized data. Thanks for checking!